packages/server/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@webcomponents/internal-site-server",
```

Comment:
I would probably name the folder `site-server` to be more clear when browsing the FS and also seeing `catalog-server`

Reply:
Done